03/app.js

```diff
@@ -0,0 +1,14 @@
+// pobieranie buttonów 
+const buttons = document.querySelectorAll("button")
+
+
+function changeText (change) {
+    change.target.textContent = "clicked"
+    console.log("clicked");
+
+    change.target.removeEventListener('click', changeText )
+}
+
+buttons.forEach(button => {
```

Comment:
Tutaj nie trzeba sprawdzać bo mamy `querySelectorAll`, który zwraca pustą tablicę jeśli nic nie wyszuka, a to nie spowoduje błędów/problemów.